packages/dds/tree/src/feature-libraries/editable-tree/fence.json

```
@@ -0,0 +1,5 @@
+{
+    "tags": [ "forest-proxy" ],
+    "exports": ["index"],
+    "imports": ["schema-stored","tree","dependency-tracking","forest","util","rebase","feature-libraries"]
```

I don't think this should depend on rebase or feature-libraries.

```suggestion
    "imports": ["schema-stored","tree","dependency-tracking","forest","util"]
```